pubspec.yaml

```diff
@@ -5,12 +5,11 @@ homepage: https://github.com/google/process.dart
 
 dependencies:
   file: '^5.0.0'
-  meta: ^1.1.2
   path: ^1.5.1
   platform: '>=1.0.1'
 
 dev_dependencies:
   test: ^1.0.0
 
 environment:
-  sdk: '>=2.0.0 <3.0.0'
+  sdk: '>=2.9.0 <3.0.0'
```

Comment:
Not `2.10.0`?

Reply:
Ahh right, we switched versions. 😄

Reply:
actually we're currently at 2.9 and the next version is null safety. I think flutter is on an allowlist to specify a 2.10 constraint which this package isn't on yet

Reply:
ack